web/views/email/style/bulkban.text.tmpl

```diff
@@ -0,0 +1,19 @@
+{{ template "email/greeting.text" . }}
+
+{{ template "email/noticeaction.text" . }}
+
+Some of your styles have been removed from our platform for the following reason:
+{{ .Log.Reason }}
+
+Styles that were removed:
+{{ range .Styles -}}
+- {{ .Name }}
+{{ end }}
+
+{{ with .Log.Message }}	Additional message from the moderator: {{ . }}{{ end }}
```

Comment:
Is the tab here intentional?

Reply:
Nope, it migrated from the message for the regular style removal email like that when I was trying to get rid of those empty lines in the rendered code. I didn't have highlight for wrong tab/spaces enabled in the editor. Will fix that later.